CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # TensorRT OSS Release Changelog
 
+## [22.04](https://github.com/NVIDIA/TensorRT/releases/tag/22.04) - 2022-04-13
+### Added
+- TensorRT Engine Explorer v0.1.0 [README](tools/experimental/trt-engine-explorer/README.md)
+- Detectron 2 Mask R-CNN R50-FPN python [sample](samples/python/detectron2/README.md)
+- Model export script for sampleOnnxMnistCoordConvAC
+
+### Changed
+- Updated base TensorRT version to 8.2.4.2
+- Updated copyright headers with SPDX identifiers
+- Updated onnx-graphsurgeon v0.3.17 [CHANGELOG](tools/onnx-graphsurgeon/CHANGELOG.md)
+- `PyramidROIAlign` plugin refactor and bug fixes
+- Fixed `MultilevelCropAndResize` crashes on Windows
+- [#1583](https://github.com/NVIDIA/TensorRT/issues/1583) - sublicense ieee/half.h under Apache2
+- Updated demo/BERT performance tables for rel-8.2
+- Fix python hangs at IndexErrors when when TF is imported after TensorRT
```

Comment:
Can this issue be mentioned here? https://github.com/NVIDIA/TensorRT/issues/1774. Also remove duplicate "when".
```suggestion
- [#1774](https://github.com/NVIDIA/TensorRT/issues/1774) Fix python hangs at IndexErrors when TF is imported after TensorRT
```